app/javascript/stylesheets/dashboard.scss

```diff
@@ -49,3 +49,21 @@
 ul.collections {
   list-style-type: none;
 }
+
+#workTypeModal {
+  a {
+    text-decoration: none;
+
+    &:after {
+      font-family: Fontawesome;
+      content: '\f078';
+      padding-left: 0.5rem;
+    }
+
+    &.collapsed {
+      &:after {
+        content: '\f054';
+      }
+    }
```

Comment:
This varies the chevron appearance to the right of the "See more/fewer options" link (see screencast)